src/common/components/addressForm/addressForm.tpl.html

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col-sm-12">
-    <div class="form-group is-required" ng-class="{'has-error': ($ctrl.parentForm.addressCountry | showErrors) || $ctrl.loadingCountriesError}">
+    <div ng-if="$ctrl.useV3 !== 'true'" class="form-group is-required" ng-class="{'has-error': ($ctrl.parentForm.addressCountry | showErrors) || $ctrl.loadingCountriesError}">
```

Comment:
We need to make sure this defaults properly to US without the field being here. I'm still not sold on this change from the stakeholders.

Reply:
If it's defaulting to the US, why remove it, as many users won't need to engage with it? Or is it that the stakeholders want to remove any field that could be unnecessary?